hashids.cfc

```diff
@@ -159,7 +160,7 @@ component output="false" persistent="false" {
 		var sepDiff = alphabetLen / sepsLen;
 
 		if( !len(seps) || (sepDiff > getSepDiv()) ){
-			sepsLen = ceiling(sepDiff);
+			sepsLen = ceiling(alphabetLen / getSepDiv());
```

Comment:
This change breaks the test_specifying_custom_hash_alphabet. This test is to make sure the output is compatible with the Java version of the implementation:

https://github.com/jiecao-fm/hashids-java/blob/master/src/test/java/fm/jiecao/lib/HashidsTest.java